### ISSUE TITLE
Boot stopped at [EB|`B:WFDW] Err(0xE), 0 @ LocHB 71B4903C-14EC-42C4-BDC6-CE1449930E49

### ISSUE THREAD

**Issue by Lod3** [myspaghetti/macos-virtualbox#385]
Tried all CPU profiles, no change. 
Currently stuck at [EB|`B:WFDW] Err(0xE), 0 @ LocHB 71B4903C-14EC-42C4-BDC6-CE1449930E49 error.

**Comment by myspaghetti** [comment]
Hi! Please execute `./macos-guest-virtualbox.sh troubleshoot` and attach the resulting `macOS_troubleshoot.txt` file here.

**Comment by KibSquib48** [comment]
hey, I'm having the same issue, here's my macOS_troubleshoot.txt
[macOS_troubleshoot.txt](https://github.com/myspaghetti/macos-virtualbox/files/5825296/macOS_troubleshoot.txt)

**Comment by myspaghetti** [comment]
@KibSquib48 macOS isn't designed to run on a Pentium/Atom processor so if you're adamant about trying, you need to manually configure the VirtualBox CPUID settings. There are reports on the internet of successfully booting macOS with OpenCore with specific CPUID settings and a kernel patch. This is outside the scope of the script.

**Comment by myspaghetti** [comment]
Closing until further information is provided.

**Comment by Lod3** [comment]
Gave up, thanks for the effort. Going to use an old Macbook Air

**Comment by ctsrc** [comment]
Hi I have the same issue. I tried a couple of different CPU profiles, but each time I boot the VM from the VirtualBox GUI I get the same error message that is described by the creator of this issue.

First I tried shutting down the VM and running

```zsh
VBoxManage modifyvm macOS --cpu-profile host
VBoxManage modifyvm macOS --cpuidremoveall
```

and starting the VM again. Same error.

Then tried shutting down the VM and running

```zsh
VBoxManage modifyvm macOS --cpu-profile "Intel Core i7-5600U"
```

and starting the VM again. Same error.

Then tried stopping the VM, running

```zsh
VBoxManage modifyvm macOS --cpu-profile "Intel Core i7-6700K"
```

and starting the VM. Same error.

But then I exited the VirtualBox GUI and the next time I tried with a different CPU and started the VM from the commandline instead of via the GUI.

```zsh
VBoxManage modifyvm macOS --cpu-profile "Intel Xeon X5482 3.20GHz"
VBoxManage startvm macOS
```

And this time it booted, and after a couple of minutes I'm at the familiar macOS install screen.

So I'm going to try again with the other CPU profiles too and will report back in a moment, in order to figure out whether trying to start the VM from the GUI is also part of the problem or not.

**Comment by ctsrc** [comment]
So yeah, this time I ran the script again and had it remove the first VM. When it got to the error, I closed the VM window and selected shut down the VM as action to perform. With the script still waiting for me to "press enter when the Language window is ready", I opened another terminal.

In the other terminal I now ran

```zsh
VBoxManage modifyvm macOS --cpu-profile "Intel Core i7-6700K"
VBoxManage startvm macOS
```

Same error. Ok so then I shut down the VM, and went for the one that worked for me previously.

```zsh
VBoxManage modifyvm macOS --cpu-profile "Intel Xeon X5482 3.20GHz"
VBoxManage startvm macOS
```

And waited for it to boot and to get to the language selection screen. This takes a couple of minutes. Once it got to that stage I went back to the terminal where the script is running and pressed Enter like it asked me to.

I went through the rest of the steps, including "Press enter when the macOS Utilities window is ready." and "Press enter when the Terminal command prompt is ready.", followed by the script rebooting the machine and waiting for Language window to appear again which took another couple of minutes.

So far all is looking good.
